server/Tools/Jwt.cs

```diff
@@ -0,0 +1,122 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+
```

Comment:
sadly not using my algorithm :...(

Reply:
bc it does not make sense to use it, add more unnecessary complexity for this project.